html/webappapis/dynamic-markup-insertion/opening-the-input-stream/history.window.js

```diff
@@ -0,0 +1,25 @@
+// Historically, document.open() created an entry in the session history so
+// that the original page could be seen by going back. Test that this behavior
+// no longer occurs.
```

Comment:
We should add a comment saying why this doesn't use an iframe. It's totally fine if the comment is "This could probably use an iframe, but I just wrote a lot of iframe tests, so for variety let's test window.open(). If the difference matters we can convert to or add an iframe test."